test/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:7
+FROM oraclelinux:7
```

Comment:
in the kitchen-dokken driver we switched to almalinux-9. Should probably do the same here for consistency and future proofing. Then the yum commands can be updated to dnf.

Reply:
I updated the Dockerfile and got it building successfully, but kitchen-docker complains the container isn't listening for ssh connections.

Perhaps I need to change the CMD to run the ssh server? It wasn't obvious to me why it worked with Oracle 7 but not Alma 9.